src/ng/compile.js

```diff
@@ -1723,7 +1723,7 @@ function $CompileProvider($provide, $$sanitizeUriProvider) {
         if ((nodeName === 'a' && (key === 'href' || key === 'xlinkHref')) ||
             (nodeName === 'img' && key === 'src')) {
           // sanitize a[href] and img[src] values
-          this[key] = value = $$sanitizeUri(value, key === 'src');
+          this[key] = value = (value == null) ? value : $$sanitizeUri(value, key === 'src');
```

Comment:
Are those parens around `value == null` necessary? ;)

Reply:
No, but look how much cleaner it looks with the parens 😛

Reply:
De gustibus etc. but I'm not insisting. ;)